satpy/readers/seviri_l1b_native.py

```diff
@@ -370,13 +370,16 @@ def _read_header(self):
         self.mda['hrv_number_of_lines'] = int(sec15hd["NumberLinesHRV"]['Value'])
         self.mda['hrv_number_of_columns'] = cols_hrv
 
-        if self.header['15_MAIN_PRODUCT_HEADER']['QQOV']['Value'] == 'NOK':
-            warnings.warn(
-                "The quality flag for this file indicates not OK. "
-                "Use this data with caution!",
-                UserWarning,
-                stacklevel=2
-            )
+        if '15_MAIN_PRODUCT_HEADER' not in self.header:
+            logger.info("Quality flag check was not possible due to missing 15_MAIN_PRODUCT_HEADER.")
+        else:
+            if self.header['15_MAIN_PRODUCT_HEADER']['QQOV']['Value'] == 'NOK':
```

Comment:
elif?

Reply:
sure, changed to elif (in "add test" commit, message wrong by mistake, sorry)